src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -1265,7 +1266,7 @@ def _lazy_read_ca_bundle(self):
                     self.read_cert_bundle(ca_bundle)
                 else:
                     import sys
-                    from requests import certs
+                    from .vendored.requests import certs
```

Comment:
`generic_requests` ?

Reply:
or if not then, the above `from .vendored import requests as generic_requests` is required?

Reply:
I think this was meant to reach out to the ocsp responders directly. @sfc-gh-mkeller we'd want to use regular requests outside the vendor-d package for this one.

Reply:
Oops, thank you @sfc-gh-hchaturvedi